src/main/kotlin/org/opensearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -790,13 +806,30 @@ class IndexReplicationTask(id: Long, type: String, action: String, description:
             ShardReplicationExecutor.TASK_NAME, replicationParams)
     }
 
+    override fun onIndexShardClosed(shardId: ShardId, indexShard: IndexShard?, indexSettings: Settings) {
+    }
+
```

Comment:
Can we add a comment as to why this override exists?

Reply:
Done